yetanotherco/aligned_layer
# Feat(contracts): Add script + guide to set aggregator address

# Add script + guide to set aggregator address
## Description

 Adds script and guide to set the aggregator address of the AlignedServiceManager.sol contract

### To Test
- Start local anvil node.
```bash
make anvil_start_with_block_time
```bash
- Check the current value of `alignedAggregator` within the AlignedServiceManager.sol contract.
```bash
cast call 0x851356ae760d987E095750cCeb3bC6014560891C "alignedAggregator()(address)"
```
- Set the value of `alignedAggregator` using the script.
```bash
export AGGREGATOR_ADDRESS=0x0000000000000000000000000000000000000001
make set_aggregator_address_devnet
```
- Verify `alignedAggregator` has been changed to `0x0000000000000000000000000000000000000001` within the AlignedServiceManager.sol contract.
```bash
cast call 0x851356ae760d987E095750cCeb3bC6014560891C "alignedAggregator()(address)"
```

## Type of change

Please delete options that are not relevant.


## Checklist

  - [Link to the open issue addressing it]()
